src/constants.ts

```diff
@@ -12,6 +13,7 @@ export const newLineCharacters = '\n\n';
  * where the not matched imports should be placed
  */
 export const THIRD_PARTY_MODULES_SPECIAL_WORD = '<THIRD_PARTY_MODULES>';
+export const BUILTIN_MODULES = builtinModules.join('|');
```

Comment:
Just a bit of warning, this regex needs to be a bit more strict, see https://github.com/IanVS/prettier-plugin-sort-imports/pull/17 for the implementation in a fork.